src/app/static/src/js/reportLog.ts

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue';
```

Comment:
Is this entire file necessary? I might be wrong, but it looks like the `reportLog` component is already mounted in `runReportTabs`

Reply:
I don't think it's necessary because I am working on the details. This ticket is just meant to be a component for runningReportDetails to mount.